db/migrate/20171027225408_create_indexes.rb

```diff
@@ -0,0 +1,33 @@
+class CreateIndexes < ActiveRecord::Migration[5.0]
+  def change
+    create_table :indexes do |t|
+      t.integer :user_id, null: false
+      t.citext :name, null: false
+      t.text :description
+      t.integer :privacy, null: false, default: 0
+      t.boolean :open_to_anyone, null: false, default: false
```

Comment:
I assume this (`open_to_anyone`) is about editing privileges instead of just a regular privacy option? I might go for something like `community_editable` to stop it from being confused?

Reply:
Yeah, it's about "who can add things to this index", I had named it this to be consistent with boards (and in case I ever made it a non-boolean and built in an IndexAuthors structure, so that the name wouldn't change) but if you think it's confusing I can make it community_editable, sure

Reply:
… Oh, good point, `open_to_anyone` is about people who can add to the board on a regular board anyway. This should be fine.